scripts.v3/generate.js

```diff
@@ -45,6 +46,12 @@ const yargs = require('yargs')
         example: '../dist/snapshot',
         demandOption: false
     })
+    .option('publish', {
+        type: 'boolean',
+        default: false,
```

Comment:
@randiratnayake, I have to ask you to changes this `default: false` setting to `true`, otherwise this will force many people to update their pipelines.

Reply:
@azaslonov Sorry, I am confused. It is the current behaviour not calling publish after import. By defaulting the value to 'false' and making the parameter not mandatory, I am honouring the current usage as is. I have tested this.

By adding this parameter and giving a choice to the developer, new implementations can avoid custom workarounds in the pipelines to publish API-M after import.

Reply:
Sorry, you're right, looks like I confused this script with migrate.js.